src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -275,6 +278,10 @@ private int parseCvrFile(
       String precinctPortion = this.precinctPortions.get(precinctPortionId);
       String ballotTypeId = adjudicatedData.get("BallotTypeId").toString();
 
+      if (precinct == null && precinctPortion != null) {
+        precinct = precinctPortion;
+      }
```

Comment:
Based on the comments on line 261, this is a workaround to have `precinct`s populated for older files that only has `precinctPortion`